yoyodyne/models/base.py

```diff
@@ -294,7 +294,7 @@ def validation_step(
             greedy_predictions, 2, 0, target_padded.size(1)
         )
         loss = self.loss_func(greedy_predictions, target_padded)
-        return {"val_accuracy": accuracy, "val_loss": loss}
+        return {"val_correct": correct, "val_total": total, "val_loss": loss}
```

Comment:
does this end up logged on the TQDM slider thingy? if so can we suppress it?

Reply:
It doesn't. Only if we call `self.log`